src/main/steps/c100-rebuild/check-your-answers/CheckYourAnswersGetController.test.ts

```diff
@@ -33,4 +37,1095 @@ describe('DocumentUpload Get Controller', () => {
 
     expect(req.session.userCase.caseId).toEqual('1111');
   });
+
+  test('checkYourAnswerFlow1', async () => {
+    const reqs = checkYourAnswerFlow1;
+    await controller.get(reqs, res);
+    expect(reqs.session.userCase).toEqual({
+      caseId: 1669809594776886,
```

Comment:
@niranjan-301 We can use the data defined in the request files for comparison, no need to add it again here.